src/pages/store/index.vue

```diff
@@ -662,15 +662,19 @@ export default {
     },
 
     bookstoreItems() {
-      const bookstoreItemsClassIdsSetFromCMS = new Set(
-        this.bookstoreItemsFromCMS.map(item => item.classId)
-      );
-      return [
+      const items = [
         ...this.bookstoreItemsFromCMS,
-        ...this.nftBookstoreLatestItems.filter(
-          item => !bookstoreItemsClassIdsSetFromCMS.has(item.classId)
-        ),
+        ...this.nftBookstoreLatestItems,
       ];
+      const uniqueIds = new Set();
+      const dedepedItems = [];
```

Comment:
```suggestion
      const dedupedItems = [];
```

What was the bug though